config/v1/types_feature.go

```diff
@@ -173,7 +173,6 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 		with(gateGatewayAPI).
 		with(maxUnavailableStatefulSet).
 		without(eventedPleg).
-		with(sigstoreImageVerification).
```

Comment:
@JoelSpeed should this line be deleted?

Reply:
The original intention was to use the TechPreviewNoUpgrade.  I was wondering the https://github.com/openshift/api/pull/1457#discussion_r1374461722 means it can not use TechPreviewNoUpgrade.

> If you want to use CustomNoupgrade and TechPreviewNoUpgrade, then it should be left in

Now I think I misunderstood the previous comment, you were suggesting there should be both 0000_10_config-operator_01_clusterimagepolicy-CustomNoUpgrade.crd.yaml and 0000_10_config-operator_01_clusterimagepolicy-TechPreviewNoUpgrade.crd.yaml.

Can we use TechPreviewNoUpgrade only?